README.md

```diff
@@ -101,7 +101,7 @@ These also work with functional components.
 Here is our MostRecentSearch component again, rewritten as a class with `subscribeToStateUpdates`.
```

Comment:
out of scope here but I made an item to mark subscribeToStateUpdates as deprecated too. This came up a few sprints ago when we had an item to write unit tests for it, which felt a little silly since in the component lib and a bunch of sample sites both our team didn't use it a single time. afaik product has also never used it
https://yexttest.atlassian.net/browse/SLAP-2208

Reply:
is this up to date with develop branch? that item is done, and the README.md is updated to remove `subscribeToStateUpdates` portion already in develop

Reply:
should I merge develop into this branch then?